src/stores/FileBrowserStore.ts

```diff
@@ -78,6 +105,9 @@ export class FileBrowserStore {
         if (AppStore.Instance.activeFrame && mode === BrowserMode.SaveFile) {
             this.saveFilename = AppStore.Instance.activeFrame.frameInfo.fileInfo.name;
         }
+        this.updateIniSaveSpectralRange();
```

Comment:
could we not use a `@computed` value (`automaticSpectralRange` instead of this and an autorun?)
I don't like `IniSaveSpectralRange`. If `Ini` is short for `Initial`, rather use `Init` (or just use `Initial`)

Reply:
There are several situations we need to initialize SaveSpectralRange: activeFrame changes, updateSpectralSystem, updateSpectralCoordinate, and start a new FileBrowserDialog. Suppose to use @reaction and @action to trigger the update of SaveSpectralRange.

Reply:
Changed the naming